wordle-web/pages/lastTenDays.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <container class="game-container" fluid>
+    <div id="statsContainer"></div>
+  </container>
+</template>
+
+<script setup lang="ts">
+import Axios from "axios"
+  // Function to generate HTML for each day
+  function generateStatsHTML(data) {
+    let html = '';
+    data.forEach(day => {
+      html += `
+        <div class="stats-card">
+          <h3>${day.date}</h3>
+          <p>Total Times Played: ${day.totalTimesPlayed}</p>
+          <p>Total Wins: ${day.totalWins}</p>
+          <p>Total Losses: ${day.totalLosses}</p>
+        </div>
+      `;
+    });
+    return html;
+  }
+
+  // Get container element
+  const statsContainer = document.getElementById('statsContainer');
+
+  // Make API call using Axios
+  const date = '5-20-2024'; // Or use any other date
+  Axios.get(`Game/LastTenWordStats/${date}`)
+    .then(response => {
+      const data = response.data;
+      // Generate and append HTML to container
+      statsContainer.innerHTML = generateStatsHTML(data);
```

Comment:
so from inspecting the page and trying to see why it is not loading it is this line statsContainer.innerHTML = generateStatsHTML(data) that is giving an error. I am not sure if Direct DOM manipulation is needed here when we can use vuetify reactive and onMounted for this

Reply:
Completely changed this, thank you for this insight though!